addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -147,6 +147,17 @@ const uniqueRequestId = () => Math.floor(Math.random() * 1000 * 1000 * 1000);
  */
 const buildEmptyResponse = () => new Response(JSON.stringify({ jsonrpc: "2.0", id: uniqueRequestId(), result: {} }));
 
+/**
+ * Fetch the request while adding the prefetch header.
+ * @param request
+ * @returns {Promise<Response>}
+ */
+const fetchWithPrefetchHeader = async request => {
+    const headers = new Headers();
+    headers.append("X-Prefetch", "1");
+    return fetch(request, { headers: headers });
```

Comment:
Styling tip: you can avoid the key/val naming redundancy when the key == value's variable:

```suggestion
    return fetch(request, { headers });
```

Reply:
Thanks for the tip; it is changed.